workspaces/sonarqube/.changeset/proud-jobs-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-sonarqube-backend': minor
+---
+
+Switch from basic auth to recommended bearer auth to support SonarQube Cloud too.
```

Comment:
Can we add a link to the docs showing that it's recommend for both on-prem and cloud, please?

Reply:
Done, should be good now.